### DISCUSSION TITLE
Valid `later` syntax is marked as invalid by renovate

### DISCUSSION THREAD

**Discussion by billy-mosse** [renovatebot/renovate#18055] [Request Help]
Hi! I have a tiny js program to test some `later` schedules before using them in renovate. I tested `every 1 month on the last day of the month before 12pm CET starting 9/30/2022`, and it worked fine, but renovate is marking it as invalid.

My questions would be:

1. Is this expected?
2. Is maybe "starting on" schedules not supported by renovate? What else is supported by `later` but not by renovate? 

Thank you

**Comment by viceice** [comment]
what's the error message renovate shows?

**Comment by rarkins** [comment]
I checked it using https://codepen.io/rationaltiger24/full/ZExQEgK and it thinks its valid.

Another possibility: always use cron if you can. It will generally be a lot easier to be sure of than `later`
